send.go

```diff
@@ -108,9 +104,6 @@ func addAddress(list []string, addr string) []string {
 }
 
 func parseAddress(field string) (string, error) {
-	addr, err := mail.ParseAddress(field)
-	if err != nil {
-		return "", fmt.Errorf("gomail: invalid address %q: %v", field, err)
-	}
-	return addr.Address, nil
+
+	return field, nil
```

Comment:
Only need to skip the "@" check, all other errors should be kept!

Reply:
Done